packages/components/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,105 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { cloneElement, forwardRef, type MouseEvent, type ReactElement, type ReactNode } from 'react';
+
+import { buttonIconSizeMap, buttonStyles } from './styles/buttonStyles';
+import { iconStyles } from './styles/iconStyles';
+import { textStyles } from './styles/textStyles';
+
+export type ButtonRounding = 'medium' | 'full';
+
+export type ButtonStyle = 'default' | 'positive' | 'negative' | 'danger' | 'loud';
+
+export type UpdatedButtonStyle = 'default' | 'positive' | 'negative' | 'danger' | 'loud';
+
+export type ButtonSize = 'small' | 'medium' | 'large';
+
+export type ButtonType = 'button' | 'submit' | 'reset';
+
+export type ButtonEmphasis = 'default' | 'weak' | 'strong';
+
+export type ButtonProps = {
+    type?: ButtonType;
+    title?: string;
+    style?: ButtonStyle;
+    emphasis?: ButtonEmphasis;
+    hideLabel?: boolean;
+    size?: ButtonSize;
+    rounding?: ButtonRounding;
+    disabled?: boolean;
+    icon?: ReactElement;
+    children?: ReactNode;
+    onClick?: (event?: MouseEvent<HTMLButtonElement>) => void;
+    hugWidth?: boolean;
+    'aria-label'?: string;
+    'aria-describedby'?: string;
+    formId?: string;
+    'data-test-id'?: string;
+};
+
+export const ButtonComponent = (
+    {
+        type = 'button',
+        style = 'default',
+        size = 'medium',
+        rounding = 'medium',
+        emphasis = 'strong',
+        title = '',
+        hideLabel = false,
+        disabled = false,
+        icon,
+        children,
+        onClick,
+        hugWidth = true,
+        formId,
+        'aria-label': ariaLabel,
+        'aria-describedby': ariaDescribedBy,
+        'data-test-id': dataTestId = 'button',
+    }: ButtonProps,
+    ref: React.ForwardedRef<HTMLButtonElement | null>,
```

Comment:
```suggestion
    ref: ForwardedRef<HTMLButtonElement | null>,
```
We don't use the React namespace

Reply:
yep, oversight from my side.

Removed